cmake/create_test.cmake

```diff
@@ -214,20 +256,38 @@ function(ginkgo_create_common_test_internal test_name exec_type exec)
     if(exec IN_LIST common_test_DISABLE_EXECUTORS)
         return()
     endif()
+    if (exec STREQUAL reference)
+        set(test_resource_type "")
```

Comment:
Doesn't this prevent the reference test to participate in the resource allocation?

Reply:
The reference tests don't have resources assigned, so they run with the maximal amount of parallelism that `-j` specifies.

Reply:
Doesn't that lead to oversubscription, at least in the omp case? Or is it guarenteed that all the reference+core test will be done before the omp/device tests?

Reply:
Right, I missed that. So in theory yes, in practice the reference and core tests are really fast (except for maybe FFT and ISAI) and only use a single core, so at most we have a bit of oversubscription on the OpenMP execution, but since those also tend to spend more time in reference kernels than OpenMP kernels (since they are hopefully more efficient), I am not sure this matters much to us?

Reply:
I guess we can just leave it and see if that actually makes any issues.